### DISCUSSION TITLE
Error Running lime

### DISCUSSION THREAD

**Discussion by patch-357** [HaxeFlixel/flixel#2373] [Q&A]
i am trying to get lime to run and i keep getting the same error

i have installed lime properly but when i go to run it it get: 
Error: Library lime is not installed : run 'haxelib install lime'
but if i go to install it again it says:
You already have lime version 7.9.0 installed
Error: [file_contents,C:\HaxeToolkit\haxe\lib\lime/.current]

what do i do to fix this
i am new to coding and confused

**Comment by RichardBray** [comment]
Looks like the path to your lime executable might be incorrect. Are you using VS Code?

**Reply by patch-357** [reply]
Like I said, I'm new to code
What is VS code?
Sorry if it's something simple

**Reply by patch-357** [reply]
i fixed that one issue, but now i have another
C:\Users\Madison\Documents\funkin-windows-64bit>lime test windows -release"
C:/Users/Madison/lime/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/Users/Madison/lime/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
source/PlayState.hx:150: characters 3-39 : Warning : FlxCamera.defaultCameras is deprecated, use FlxG.cameras.setDefaultDrawTarget instead
C:\Users\Madison\Documents\funkin-windows-64bit\export\release\windows\obj>setlocal enabledelayedexpansion
Error: 64bit is not automatically supported for this version of VC. Set HXCPP_MSVC_CUSTOM and manually configure the executable, library and include paths
Error: Error: 64bit is not automatically supported for this version of VC. Set HXCPP_MSVC_CUSTOM and manually configure the executable, library and include paths

**Comment by plasticocelot123** [comment]
VS Code is a notepad (Of a sort) that is built to edit and autocorrect code. It is made by Microsoft. Get it here: [](https://code.visualstudio.com/Download)

**Reply by plasticocelot123** [reply]
https://code.visualstudio.com/Download
